static/src/stylesheets/module/commercial/_advert.scss

```diff
@@ -39,3 +42,64 @@
 .advert__more {
     margin-top: $gs-baseline / 2;
 }
+
+.advert--small > .advert__title {
+    padding-top: $gs-baseline / 2;
+}
+
+@include mq(tablet) {
+    .advert--large {
+        flex-direction: row;
+        align-items: stretch;
+        padding-bottom: 0;
+
+        > .advert__text {
+            flex: 1;
+            padding: $gs-baseline / 2 $gs-gutter / 2 $gs-baseline / 2 0;
+        }
+
+        > .advert__image-container {
+            order: 0;
+            margin-bottom: 0;
+            margin-left: $gs-gutter / 2;
+        }
+
+        .advert__image {
+            width: 100%;
+        }
+    }
+
+    .advert--large--inverse {
```

Comment:
This is smelly. Can't it be `advert--large` and `advert--inverse` and you can mix and match in the template?

When I see modification of the modification, usually there is always a different way how to approach it. And it is same across the whole file.